library-best-practices/README.md

```diff
@@ -9,6 +9,10 @@ made. We have found that following these rules will result in higher quality
 Java libraries with fewer dependency conflicts and other kinds of problems. The
 list is open-ended, so new ones may be added from time to time.
 
+*Note* This list is currently in "Beta," which means that we don't anticipate
```

Comment:
I'd go stronger: This list is in alpha. It's an early draft and we're still developing it.

Reply:
Done in https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/191